.github/workflows/push.yml

```diff
@@ -12,17 +12,15 @@ jobs:
     - name: operator-sdk
       uses: ./.github/action/operator-sdk
       with:
-        args: build lightbend-docker-registry.bintray.io/lightbend/akkacluster-operator:latest
+        args: build lightbend-docker-registry.bintray.io/lightbend/akkacluster-operator-certified:latest
     - name: bintray login
-      uses: actions/docker/login@8cdf801b322af5f369e00d85e9cf3a7122f49108
+      run: docker login
```

Comment:
I see you trimmed down the login command...  Was that intentional or is a rebase in order?

Reply:
This is a PR into `certified`. This was intentional. The rebase wasn't worth the effort here.